lib/orcid/add_works.rb

```diff
@@ -41,6 +41,9 @@ def add_work(author, contribution, orcid_user)
       contribution.save!
       logger&.info("#{self.class} - author #{author.id} - added publication #{contribution.publication.id} with put-code #{contribution.orcid_put_code}")
       true
+    rescue Orcid::PubMapper::PubMapperError => e
```

Comment:
PubMapperErrors are worthy of logging, but not notifying.

Reply:
so is the idea that we monitor the logs for a bit to see the types of unexpected mapping issues that are occurring?

Reply:
We should keep in eye out, but this is primarily going to report issues like "this publication doesn't have an identifier." Any significant issues will be HB alerted.